website/mails/listeners.py

```diff
@@ -44,6 +44,14 @@ def queue_first_public_project_email(user, node, meeting_creation):
                 osf_support_email=settings.OSF_SUPPORT_EMAIL,
             )
 
+
+@project_signals.privacy_set_public.connect
```

Comment:
This signal is only sent when `set_privacy` is called with auth (which doesn't happen on sanction completion), so DOI generation is broken right now.